docs/source/cpp/compute.rst

```diff
@@ -348,6 +348,22 @@ Bit-wise functions
   out of bounds for the data type.  However, an overflow when shifting the
   first input is not error (truncated bits are silently discarded).
 
+Rounding functions
+~~~~~~~~~~~~~~~~~~
+
+Rounding functions convert a numeric input into an approximate value with a
+simpler representation based on the rounding strategy.
```

Comment:
This is worded a little confusingly in my opinion. If we're going to reference rounding strategy here, the notes column should describe the rounding behavior for each function (even if it's just the 'obvious' or 'expected' one).

Or alternatively, something like 'rounding functions find the nearest integer (as a floating-point value) to the argument based on a rounding strategy'.

Reply:
I added the rounding functions section/text based on the soon-to-be-ready [`round` and `mround` functions](https://github.com/apache/arrow/pull/10349). Although, `floor`, `ceil`, and `trunc` do round to the nearest integer, `round/mround` do not necessarily. They have options to specify fractional precision and have options for various rounding strategies.

Reply:
Ah, ok, sounds good.